ReactCommon/react/renderer/graphics/platform/ios/RCTPlatformColorUtils.mm

```diff
@@ -179,6 +179,9 @@
 static inline facebook::react::ColorComponents _ColorComponentsFromUIColor(RCTUIColor *color) // [macOS]
 {
   CGFloat rgba[4];
+#if TARGET_OS_OSX // [macOS
+  color = [color colorUsingColorSpace:[NSColorSpace genericRGBColorSpace]];
+#endif // macOS]
```

Comment:
Wouldn't iOS have this issue too since they too have platform colors? Would this be an upstreamable fix to an extent?

Reply:
Good call. I'll ask the RN iOS team if they encountered the same issue.